examples/wakaama/data/bootstrap.json

```diff
@@ -0,0 +1,91 @@
+{
+  "testRIOTDevice": {
+    "servers": {
+      "0": {
+        "shortId": 10,
+        "lifetime": 20,
+        "defaultMinPeriod": 1,
+        "notifIfDisabled": true,
+        "binding": "U"
+      }
+    },
+    "security": {
+      "0": {
+        "uri": "coaps://localhost:5685",
```

Comment:
Can you clarify the difference between "0" (false?) and "1" ?

Also, what is the difference when using the leshan-server and the leshan-bssserver?

Reply:
> Can you clarify the difference between "0" (false?) and "1" ?

Those are the indexes of the entry in the security "array".

"security.0" is the bootstrap server itself, so it defines which keys the client needs to know to establish a DTLS channel with the bootstrap server (note the `"bootstrapServer": true,` below the URI).

"security.1" is associated to "servers.0" defined above via its *shortId*.

Basically, the "servers" objects defines a list of servers to install on the client and the "security" objects defines their connection details, including keys.

They keys for the bootstrap server and the normal servers should usually not be the same, as that sorta defeats the point of having this bootstrap mechanism to begin with (see below), but for testing I made them the same, so the config might look a bit confusing.

> Also, what is the difference when using the leshan-server and the leshan-bssserver?

The only function of a bootstrap server is to give a connecting device a list of "normal" servers to register with, including the required key material.